src/AutoBuild.jl

```diff
@@ -1268,79 +1278,78 @@ function build_jll_package(src_name::String, build_version::VersionNumber, code_
+    print_source(io, ::DirectorySource) = nothing
+    function print_jll(io, dep)
+        depname = getname(dep)
+        if from_yggdrasil
+            # In this case we can easily add a direct link to the repo
+            println(io, "* [`", depname, "`](https://github.com/JuliaBinaryWrappers/", depname, ".jl)")
+        else
+            println(io, "* `", depname, ")`")
+        end
+    end
+    print_product(io, p::Product) = println(io, "* `", typeof(p), "`: `", variable_name(p), "`")
     # Add a README.md
     open(joinpath(code_dir, "README.md"), "w") do io
         print(io,
               """
-              # $(src_name)_jll.jl
-
-              This is an autogenerated package constructed using [`BinaryBuilder.jl`](https://github.com/JuliaPackaging/BinaryBuilder.jl).
-
-              For more details about JLL packages, see `BinaryBuilder.jl` [documentation](https://juliapackaging.github.io/BinaryBuilder.jl/dev/jll/).
-
-              """,
-              iszero(length(keys(products_info))) ? "" :
-              """
-              ## Products
-
-              The code bindings within this package are autogenerated from the following `Products` defined within the `build_tarballs.jl` file that generated this package:
-
-              ```julia
-              products = [
-              $(join(collect("    $(typeof(p))($(product_names(p)), :$(variable_name(p)))" for (p, _) in products_info), ",\n"))
-              ]
-              ```
-
-              """,
-              """
-              ## Usage example
-
-              For example purposes, we will assume that the following products were defined in the imaginary package `Example_jll`:
-
-              ```julia
-              products = [
-                  FileProduct("src/data.txt", :data_txt),
-                  LibraryProduct("libdataproc", :libdataproc),
-                  ExecutableProduct("mungify", :mungify_exe)
-              ]
-              ```
+              # `$(src_name)_jll.jl` (`v$(build_version)`)
 
-              With such products defined, `Example_jll` would contain `data_txt`, `libdataproc` and `mungify_exe` symbols exported. For `FileProduct` variables, the exported value is a string pointing to the location of the file on-disk.  For `LibraryProduct` variables, it is a string corresponding to the `SONAME` of the desired library (it will have already been `dlopen()`'ed, so typical `ccall()` usage applies), and for `ExecutableProduct` variables, the exported value is a function that can be called to set appropriate environment variables.  Example:
-
-              ```julia
-              using Example_jll
+              This is an autogenerated package constructed using [`BinaryBuilder.jl`](https://github.com/JuliaPackaging/BinaryBuilder.jl).""")
+        if from_yggdrasil
+            print(io, " The recipe can be found on [`Yggdrasil`](https://github.com/JuliaPackaging/Yggdrasil/), the community build tree.")
```

Comment:
Since we know the `name`, we can make an educated guess and link directly to the project folder.

Reply:
In most cases it will work, but there are some notable exception, like [`Coin-OR`](https://github.com/JuliaPackaging/Yggdrasil/tree/4a49d243c6a211eb18de52abde1ac3fdfc6a48b0/C/Coin-OR), [`OpenBLAS`](https://github.com/JuliaPackaging/Yggdrasil/tree/4a49d243c6a211eb18de52abde1ac3fdfc6a48b0/O/OpenBLAS) and [`LLVM`](https://github.com/JuliaPackaging/Yggdrasil/tree/4a49d243c6a211eb18de52abde1ac3fdfc6a48b0/L/LLVM).  [`R/Librsvg`](https://github.com/JuliaPackaging/Yggdrasil/tree/4a49d243c6a211eb18de52abde1ac3fdfc6a48b0/R/Librsvg) is also an interesting case where the letter directory doesn't match the first letter of the builder name.  These are the reason why I didn't add a direct link to the directory.  If we're ok with having a few broken special cases, I can add the link.

Reply:
I think all except `Librsvg` are actually fine, it's still more helpful than the overall Yggdrasil link.  For `Librsvg`, we may just move that to `L` sometime.  :P

Reply:
Perhaps then a CI check should also be introduced which enforces that the builder name matches the directory name, to prevent future drift?

I guess `OpenBLAS` is indeed fine, but I don't understand why `Coin-OR` is?

Reply:
Fixed in #807